addons/mail/tests/test_mail_render.py

```diff
@@ -319,3 +319,19 @@ def test_template_rendering_various(self):
                 src, partner._name, partner.ids, engine=engine,
             )[partner.id]
             self.assertEqual(result, expected)
+
+    def test_mail_template_neutralize(self):
+        """ ensure mail templates can be neutralized """
+        fake_mail_server = self.env['ir.mail_server'].create({
+            'name': "fake test email server",
+            'smtp_host': "mail.example.com",
+            'smtp_port': 15626,
+        })
+        self.test_template_jinja.mail_server_id = fake_mail_server.id
```

Comment:
Assignments are designed to receive a recordset
```suggestion
        self.test_template_jinja.mail_server_id = fake_mail_server
```